dist/angular-chips.js

```diff
@@ -219,11 +217,19 @@
                          */
                         var chipTemplates = iElement.find('chip-tmpl');
                         if (chipTemplates.length > 0 && parseInt(event.target.getAttribute('index')) - 1 === chipTemplates.length)
-                            iElement.find('chip-tmpl')[chipNavigate(37)].focus();
+                            chipNavigate === null ? focusOnChip() : iElement.find('chip-tmpl')[chipNavigate(37)].focus();
                     }
 
                 } else if (event.keyCode === 37 || event.keyCode === 39) {
-                    chipNavigate === null ? focusOnChip() : iElement.find('chip-tmpl')[chipNavigate(event.keyCode)].focus();
+                    if (chipNavigate === null) {
+                      focusOnChip();
+                      if (event.target.nodeName === 'INPUT') {
```

Comment:
I think you added this block to use the left and right arrow key in inside the input field. but this will still set the focus on the last chip. because of the focusOnChip() method on line number 225.

I think the right way is. If user press the left or right arrow from inside the input field. we should not set the focus on the last chip. if user want to set the focus on the last chip, user can press the Esc button to remove the focus from the input field and set the focus on the last chip.

so please remove this block I'll merge this PR.